debugger/src/main/resources/logback.xml

```diff
@@ -27,7 +27,17 @@
       <pattern>%date{ISO8601} [%thread] %highlight(%-5level) %cyan(%logger{15}) - %msg %n</pattern>
     </encoder>
   </appender>
-  <root level="DEBUG">
+
+  <appender name="FILE" class="ch.qos.logback.core.FileAppender">
+      <file>${LOG_FILE}</file>
+      <append>false</append>
+      <encoder>
+          <pattern>%date{ISO8601} [%thread] %highlight(%-5level) %cyan(%logger{15}) - %msg %n</pattern>
+      </encoder>
+  </appender>
+
+  <root level="${LOG_LEVEL}">
```

Comment:
Oh! I found a way to specify defaults: https://logback.qos.ch/manual/configuration.html#defaultValuesForVariables